appveyor.yml

```diff
@@ -7,12 +7,20 @@ init:
       if ($env:APPVEYOR_REPO_BRANCH -eq "dev")
       {
         $env:prereleaseTag = "{0}.{1}.{2}.{3}" -f $version.Major, $version.Minor, $version.Build, $version.Revision
+        $env:channel = "win-x64-prerelease"
+      } else {
+        $env:channel = "win-x64-stable"
       }
+      dotnet tool update -g vpk
+      $env:APPVEYOR_CACHE_ENTRY_ZIP_ARGS = "-t7z -m0=lzma -mx=9"
+
 - sc config WSearch start= auto # Starts Windows Search service- Needed for running ExplorerTest
 - net start WSearch
 
-cache:
-  - '%USERPROFILE%\.nuget\packages -> **.sln, **.csproj'  # preserve nuget folder (packages) unless the solution or projects change
+#cache:
+#  - '%USERPROFILE%\.nuget\packages -> **.sln, **.csproj'  # preserve nuget folder (packages) unless the solution or projects change
+
```

Comment:
was this change meant to be temporary? Also, shouldn't we be caching the dotnet tools folder here, as well?

Reply:
I think cache is not improving the building time....it seems to take very long time to unzip the cache...

Reply:
> it seems to take very long time to unzip the cache...

yeah I think you're right, seems to take about the same time...

On the other hand, installing vpk takes only ~7 seconds on appveyor... which is very strange, from my experience dotnet tool installs usually takes ages...